lib/manager/flux/extract.ts

```diff
@@ -10,14 +10,17 @@ import type { FluxManifest, FluxResource, ResourceFluxManifest } from './types';
 
 function readManifest(content: string, file: string): FluxManifest | null {
   if (isSystemManifest(file)) {
-    const versionMatch = regEx(/#\s*Flux\s+Version:\s*(\S+)/).exec(content);
+    const versionMatch = regEx(
+      /#\s*Flux\s+Version:\s*(\S+)(?:\s*#\s*Components:\s*(\S+))?/
```

Comment:
Might be better to use a stricter regex for the components to avoid shell command injection.

Reply:
Yes, see also my other comment.